modules/serverless-create-domain-mapping.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+//  * serverless/networking/serverless-domain-mapping.adoc
+
+[id="serverless-create-domain-mapping_{context}"]
+= Creating custom domain mapping for a service
+
+To map a custom domain name to a service, you must create a `DomainMapping` custom resource (CR).
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have created a Knative service, and own a custom domain that you want to map to that service.
++
+[NOTE]
+====
+Your custom domain must point to the DNS of the Knative cluster. Instructions for completing this step will vary based on your domain registrar.
```

Comment:
It's not entirely clear what this alludes to. Do you need a `CNAME` record here? Or does the domain need to use the cluster DNS itself for the DNS servers. For example, for `whois redhat.com`, we have these name servers:

```
   Name Server: A1-68.AKAM.NET
   Name Server: A10-65.AKAM.NET
   Name Server: A13-66.AKAM.NET
   Name Server: A16-67.AKAM.NET
   Name Server: A28-64.AKAM.NET
   Name Server: A9-65.AKAM.NET
```

Reply:
@nak3 can you clarify please. These docs are based on the upstream docs and it's not specified beyond this.

Reply:
Hi @jboxman Either is fine.  Perhaps we can say simple as `Your custom domain must point to the DNS of the OpenShift cluster.` like this?

CURRENT:
```
Your custom domain must point to the DNS of the Knative cluster. Instructions for completing this step will vary based on your domain registrar.
```

TOBE:
```
Your custom domain must point to the DNS of the OpenShift cluster.
```

Reply:
@nak3, sounds good to me, thanks!